### PR TITLE
Ignore nesting selectors.

### PR DESCRIPTION
**Pull request by evilebottnawi**
Why we use `if (isKeyframeRule(rule)) { return }` in selectors rule?
/cc @davidtheclark @jeddy3

### CODE REVIEW

**Comment by evilebottnawi** [comment]
Maybe need release, because rule non working correctly now.

**Comment by davidtheclark** [comment]
Thanks.

**Comment by jeddy3** [comment]
> Why we use if (isKeyframeRule(rule)) { return } in selectors rule?

Can you rephrase the question? I'm not sure what you mean.

> Maybe need release, because rule non working correctly now.

I'll release now.

**Comment by evilebottnawi** [comment]
@jeddy3 In some rules `selector-*` we use ` if (isKeyframeRule(rule)) { return }` check, why?

**Comment by jeddy3** [comment]
PostCSS doesn't distinguish between [keyframe rules/selectors/declaration-blocks/declarations](https://www.w3.org/TR/css3-animations/#keyframes) and normal ones. `isKeyframeRule` is used when we only want to check either keyframe things and normal things, for example when checking type selectors.

**Comment by evilebottnawi** [comment]
@jeddy3 we should add tests and check all `selectors-*` rule, right?

**Comment by jeddy3** [comment]
Yes. That would be proactive.

**Comment by jeddy3** [comment]
If you're keen, feel free to do this now. I'm off to gym and I can release when I'm back :)

**Comment by evilebottnawi** [comment]
@jeddy3 today i review as many rules as i can :+1:
